CMakeLists.txt

```diff
@@ -111,16 +110,24 @@ if(LIBCORO_FEATURE_NETWORKING)
     endif()
 endif()
 
+if(DEFINED EMSCRIPTEN)
+  add_compile_options(-fwasm-exceptions)
```

Comment:
can these be `target_` for the library? Usually isn't bet cmake practice to set things globally

Reply:
Will do

Reply:
Actually, they need to be enabled globally for all things compiled with emscripten otherwise it will fail. They really are globals.